src/gui/search/pluginselectdialog.h

```diff
@@ -31,11 +31,11 @@
 #define PLUGINSELECTDIALOG_H
 
 #include <QDialog>
+#include <QStringList>
```

Comment:
Why not QtContainerFwd?

Reply:
Because look below, there is a variable that is exactly this type.

Reply:
This is not visible in diff (but only forward declaration).